src/sentry/notifications/notifications/strategies/role_based_recipient_strategy.py

```diff
@@ -51,15 +67,11 @@ def determine_member_recipients(self) -> Iterable[OrganizationMember]:
         """
         raise NotImplementedError
 
-    def get_role_string(self, member: OrganizationMember) -> str:
-        role_string: str = roles.get(member.role).name
-        return role_string
-
     def build_notification_footer_from_settings_url(
         self, settings_url: str, recipient: User
     ) -> str:
-        recipient_member = self.get_member(recipient)
+        role = self.get_member_role(recipient)
         return (
             "You are receiving this notification because you're listed as an organization "
```

Comment:
@cathyteng17 does this text make sense anymore with the new changes?

Reply:
you have an org role through team membership so i believe this still applies. we could further specify that they have the role through a team, but i'm not sure if that's necessary